# Rosalind

Solving some problems of the **Bioinformatics Stronghold** of [http://www.rosalind.info](rosalind.info) 
cause I've nothing to do.  
(except doing my thesis, and stage, and a couple of exams...)

## List of exercise

| #  | ID   |  desc | done |
| -- | ---  | -------------------------| -- |
|001 | DNA 	| Counting DNA Nucleotides | [X] |
|002 | RNA 	| Transcribing DNA into RNA | [X] |
|003 | REVC 	| Complementing a Strand of DNA | [X] |
|004 | FIB 	| Rabbits and Recurrence Relations | [X] |
|005 | GC 	| Computing GC Content | [X] |
|006 | HAMM 	| Counting Point Mutations | [X] |
|007 | IPRB 	| Mendel's First Law | [X] |
|008 | PROT 	| Translating RNA into Protein | [X] |
|009 | SUBS 	| Finding a Motif in DNA | [X] |
|010 | CONS 	| Consensus and Profile | [X] |
|011 | FIBD 	| Mortal Fibonacci Rabbits | [X] |
|012 | GRPH 	| Overlap Graphs | [X] |
|013 | IEV 	| Calculating Expected Offspring | [X] |
|014 | LCSM 	| Finding a Shared Motif | [X] |
|015 | LIA 	| Independent Alleles | [ ] |
|016 | MPRT 	| Finding a Protein Motif | [X] |
|017 | MRNA 	| Inferring mRNA from Protein | [X] |
|018 | ORF 	| Open Reading Frames | [X] |
|019 | PERM 	| Enumerating Gene Orders | [X] |
|020 | PRTM 	| Calculating Protein Mass | [X] |
|021 | REVP 	| Locating Restriction Sites | [X] |
|022 | SPLC 	| RNA Splicing | [X] |
|023 | LEXF 	| Enumerating k-mers Lexicographically | [X] |
|024 | LGIS 	| Longest Increasing Subsequence | [X] |
|025 | LONG 	| Genome Assembly as Shortest Superstring | [ ] |
|026 | PMCH 	| Perfect Matchings and RNA Secondary Structures | [ ] |
|027 | PPER 	| Partial Permutations | [ ] |
|028 | PROB 	| Introduction to Random Strings | [ ] |
|029 | SIGN 	| Enumerating Oriented Gene Orderings | [ ] |
|030 | SSEQ 	| Finding a Spliced Motif | [ ] |
|031 | TRAN 	| Transitions and Transversions | [ ] |
|032 | TREE 	| Completing a Tree | [ ] |
|033 | CAT 	| Catalan Numbers and RNA Secondary Structures | [ ] |
|034 | CORR 	| Error Correction in Reads | [ ] |
|035 | INOD 	| Counting Phylogenetic Ancestors | [ ] |
|036 | KMER 	| k-Mer Composition | [ ] |
|037 | KMP 	| Speeding Up Motif Finding | [ ] |
|038 | LCSQ 	| Finding a Shared Spliced Motif | [ ] |
|039 | LEXV 	| Ordering Strings of Varying Length Lexicographically | [ ] |
|040 | MMCH 	| Maximum Matchings and RNA Secondary Structures | [ ] |
|041 | PDST 	| Creating a Distance Matrix | [ ] |
|042 | REAR 	| Reversal Distance | [ ] |
|043 | RSTR 	| Matching Random Motifs | [ ] |
|044 | SSET 	| Counting Subsets | [ ] |
|045 | ASPC 	| Introduction to Alternative Splicing | [ ] |
|046 | EDIT 	| Edit Distance | [ ] |
|047 | EVAL 	| Expected Number of Restriction Sites | [ ] |
|048 | MOTZ 	| Motzkin Numbers and RNA Secondary Structures | [ ] |
|049 | NWCK 	| Distances in Trees | [ ] |
|050 | SCSP 	| Interleaving Two Motifs | [ ] |
|051 | SETO 	| Introduction to Set Operations | [ ] |
|052 | SORT 	| Sorting by Reversals | [ ] |
|053 | SPEC 	| Inferring Protein from Spectrum | [ ] |
|054 | TRIE 	| Introduction to Pattern Matching | [ ] |
|055 | CONV 	| Comparing Spectra with the Spectral Convolution | [ ] |
|056 | CTBL 	| Creating a Character Table | [ ] |
|057 | DBRU 	| Constructing a De Bruijn Graph | [ ] |
|058 | EDTA 	| Edit Distance Alignment | [ ] |
|059 | FULL 	| Inferring Peptide from Full Spectrum | [ ] |
|060 | INDC 	| Independent Segregation of Chromosomes | [ ] |
|061 | ITWV 	| Finding Disjoint Motifs in a Gene | [ ] |
|062 | LREP 	| Finding the Longest Multiple Repeat | [ ] |
|063 | NKEW 	| Newick Format with Edge Weights | [ ] |
|064 | RNAS 	| Wobble Bonding and RNA Secondary Structures | [ ] |
|065 | AFRQ 	| Counting Disease Carriers | [ ] |
|066 | CSTR 	| Creating a Character Table from Genetic Strings | [ ] |
|067 | CTEA 	| Counting Optimal Alignments | [ ] |
|068 | CUNR 	| Counting Unrooted Binary Trees | [ ] |
|069 | GLOB 	| Global Alignment with Scoring Matrix | [ ] |
|070 | PCOV 	| Genome Assembly with Perfect Coverage | [ ] |
|071 | PRSM 	| Matching a Spectrum to a Protein | [ ] |
|072 | QRT 	| Quartets | [ ] |
|073 | SGRA 	| Using the Spectrum Graph to Infer Peptides | [ ] |
|074 | SUFF 	| Encoding Suffix Trees | [ ] |
|075 | CHBP 	| Character-Based Phylogeny | [ ] |
|076 | CNTQ 	| Counting Quartets | [ ] |
|077 | EUBT 	| Enumerating Unrooted Binary Trees | [ ] |
|078 | GASM 	| Genome Assembly Using Reads | [ ] |
|079 | GCON 	| Global Alignment with Constant Gap Penalty | [ ] |
|080 | LING 	| Linguistic Complexity of a Genome | [ ] |
|081 | LOCA 	| Local Alignment with Scoring Matrix | [ ] |
|082 | MEND 	| Inferring Genotype from a Pedigree | [ ] |
|083 | MGAP 	| Maximizing the Gap Symbols of an Optimal Alignment | [ ] |
|084 | MREP 	| Identifying Maximal Repeats | [ ] |
|085 | MULT 	| Multiple Alignment | [ ] |
|086 | PDPL 	| Creating a Restriction Map | [ ] |
|087 | ROOT 	| Counting Rooted Binary Trees | [ ] |
|088 | SEXL 	| Sex-Linked Inheritance | [ ] |
|089 | SPTD 	| Phylogeny Comparison with Split Distance | [ ] |
|090 | WFMD 	| The Wright-Fisher Model of Genetic Drift | [ ] |
|091 | ALPH 	| Alignment-Based Phylogeny | [ ] |
|092 | ASMQ 	| Assessing Assembly Quality with N50 and N75 | [ ] |
|093 | CSET 	| Fixing an Inconsistent Character Set | [ ] |
|094 | EBIN 	| Wright-Fisher's Expected Behavior | [ ] |
|095 | FOUN 	| The Founder Effect and Genetic Drift | [ ] |
|096 | GAFF 	| Global Alignment with Scoring Matrix and Affine Gap Penalty | [ ] |
|097 | GREP 	| Genome Assembly with Perfect Coverage and Repeats | [ ] |
|098 | OAP 	| Overlap Alignment | [ ] |
|099 | QRTD 	| Quartet Distance | [ ] |
|100 | SIMS 	| Finding a Motif with Modifications | [ ] |
|101 | SMGB 	| Semiglobal Alignment | [ ] |
|102 | KSIM 	| Finding All Similar Motifs | [ ] |
|103 | LAFF 	| Local Alignment with Affine Gap Penalty | [ ] |
|104 | OSYM 	| Isolating Symbols in Alignments | [ ] |
|105 | RSUB 	| Identifying Reversing Substitutions  | [ ] |
